docs/development_guide.md

```diff
@@ -15,11 +15,9 @@ for more information on using pull requests.
 ## Setup
 
 ### Fork the repository on Github
-Visit the
-[gcp-variant-transforms repository](https://github.com/googlegenomics/gcp-variant-transforms)
-to create your own fork of the repository. See
-[https://guides.github.com/activities/forking/](https://guides.github.com/activities/forking/)
-for more information.
+Visit the [gcp-variant-transforms repository](https://github.com/googlegenomics/gcp-variant-transforms) to create your own fork of the repository. See [https://guides.github.com/activities/forking/](https://guides.github.com/activities/forking/)
```

Comment:
Please keep each line <80 chars. Links that are longer than 80chars should be in separate lines.
There is a google style guide on .md docs somewhere as well (i'll ping you when I've found it).